src/sed/binning/binning.py

```diff
@@ -475,17 +476,12 @@ def normalization_histogram_from_timed_dataframe(
             histogram.
         bin_centers (np.ndarray): Bin centers used for binning of the axis.
         time_unit (float): Time unit the data frame entries are based on.
+        **kwds: Additional keyword arguments passed to the bin_dataframe function.
 
     Returns:
         xr.DataArray: Calculated normalization histogram.
     """
-    bins = df[axis].map_partitions(
```

Comment:
is this removed due to the updated dask version?

Reply:
I am using our optimized binning now for the timed dataframe. This is somewhat faster, does the sequential binning using the num_cores parameter, and shows the progress bar. The previous solution used the pandas cut to define bins, which requires bin edges rather than bin centers as our function.
I once checked that they produce very similar results (a very tiny difference was there, I think, because of different inclusion/exclusion of the bin edges into either left or right bin).